### PR TITLE
Create user event view

### PR DESCRIPTION
**Pull request by keyualiceli**

### CODE REVIEW

File: src/components/EventsDisplay/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    };
+  }
+
+    render() {
+        return (
+          <Element
+            className="element"
+            id="scroll-container"
+            style={{
+              height: "270px",
```
Comment (dunja-tomic) [inline src/components/EventsDisplay/index.js]: Move this over to the `styles.module.css` file!

File: src/components/MyEventsCard/index.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        <Card className={styles.card}>
+          <Card.Body className="py-3 my-3">
+            {this.props.children}
+            <Col className={styles.col}>
+              <Row>
+                <h3>{this.state.title}</h3>
+              </Row>
+              <Row>
+                <p>
+                  Something something, go to events, get points, enter raffle...
```
Comment (dunja-tomic) [inline src/components/MyEventsCard/index.js:31]: LOL

File: src/views/MyEventsPage/index.js [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+import {
+  getJwt,
+  getEmailFromJwt,
+  getNameFromJwt,
+  getRoleFromJwt,
+} from "../../utils/Cognito/index.js";
+import { getApplication } from "../../utils/API/index.js";
+
+const applicationDeadline = new Date(
```
Comment (dunja-tomic) [inline src/views/MyEventsPage/index.js]: Is this being used anywhere?

File: src/views/MyEventsPage/index.js [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+        pathname: "/login",
+      });
+    }
+  }
+
+  componentDidMount() {
+      // call API here to get actual points and events details
+      this.setState({
+        loadComplete: true,
+        points: 2,
```
Comment (dunja-tomic) [inline src/views/MyEventsPage/index.js]: I totally forgot to mention this - Corporate decided that all events are worth 1 point, so we can calculate points just by counting the number of events that the user has attended!

I think that Javascript should have an easy list manipulation function to count objects in a list with `attended == true`?

File: src/views/MyEventsPage/index.js [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+            </div>
+          )}
+          {this.state.loadComplete && (
+            <Container fluid>
+              <Row>
+                <Col sm="auto" className={styles.noPadding}>
+                  <DashboardSidebar />
+                </Col>
+                <Col className={styles.centerContent}>
+                <div style={{ color: "white" }}>
```
Comment (dunja-tomic) [inline src/views/MyEventsPage/index.js]: Throw this inline style into the `styles.module.css` file!

**Review by dunja-tomic — CHANGES_REQUESTED** [review]
This looks amazing! Just left a couple small comments, but otherwise 👍  👍

File: src/components/EventsDisplay/index.js (hunk 1 above)
Comment (keyualiceli) [reply src/components/EventsDisplay/index.js]: RESOLVED: done!

File: src/components/MyEventsCard/index.js (hunk 2 above)
Comment (keyualiceli) [reply src/components/MyEventsCard/index.js:31]: xd

File: src/views/MyEventsPage/index.js (hunk 5 above)
Comment (keyualiceli) [reply src/views/MyEventsPage/index.js]: RESOLVED: done!

File: src/views/MyEventsPage/index.js (hunk 4 above)
Comment (keyualiceli) [reply src/views/MyEventsPage/index.js]: RESOLVED: Put a .filter and .length function there now, using `eventsFromAPICall.filter((obj) => obj.attended).length,`. Let me know if that's okay. Thank you for the update and for looking it over!

File: src/views/MyEventsPage/index.js (hunk 3 above)
Comment (keyualiceli) [reply src/views/MyEventsPage/index.js]: RESOLVED: removed unused imports

**Review by dunja-tomic — APPROVED** [review]
Looks good to me!

**Pull request merged**